source/dyaml/event.d

```diff
@@ -221,7 +221,7 @@ Event documentEndEvent(const Mark start, const Mark end, const bool explicit) pu
 ///          style    = Scalar style.
 Event scalarEvent(const Mark start, const Mark end, const Anchor anchor, const Tag tag,
                   const Tuple!(bool, bool) implicit, const string value,
-                  const ScalarStyle style = ScalarStyle.Invalid) @safe pure nothrow @nogc
+                  const ScalarStyle style = ScalarStyle.Invalid) @trusted pure nothrow @nogc
```

Comment:
I am currently on my phone, but this looks pretty weird. There is already an `@trusted` lambda inside. Did you try leaving this one out?